fastqc/cloud/run_fastqc.py

```diff
@@ -0,0 +1,241 @@
+parser.add_argument("--project", required=True,
+                    help="Cloud project id to run the pipeline in")
+parser.add_argument("--disk-size", required=True, type=int,
+                    help="Size (in GB) of disk for both input and output")
+parser.add_argument("--input", required=True, nargs="+",
+                    help="Cloud Storage path to input file(s)")
+parser.add_argument("--output", required=True,
+                    help="Cloud Storage path to output file (with the .gz extension)")
+parser.add_argument("--logging", required=True,
+                    help="Cloud Storage path to send logging output")
+parser.add_argument("--poll-interval", default=0, type=int,
+                    help="Frequency (in seconds) to poll for completion (default: no polling)")
+args = parser.parse_args()
+
+# Create the genomics service
+credentials = GoogleCredentials.get_application_default()
+service = build('genomics', 'v1alpha2', credentials=credentials)
+
+# Run the pipeline
+operation = service.pipelines().run(body={
+  # The ephemeralPipeline provides the template for the pipeline
+  # The pipelineArgs provide the inputs specific to this run
+
+  # There are some nuances in the API that are still being ironed out
+  # to make this more compact.
+
+  'ephemeralPipeline' : {
+    'projectId': args.project,
+    'name': 'fastqc',
+    'description': 'Run "FastQC" on one or more files',
+
+    # Define the resources needed for this pipeline.
+    'resources' : {
+      # Specify default VM parameters for the pipeline
+      'minimumCpuCores': 1,  # TODO: remove this when the API has a default
+      'minimumRamGb': 3.75, # TODO: remove this when the API has a default
+
+      # Create a data disk that is attached to the VM and destroyed when the
+      # pipeline terminates.
+      'disks': [ {
+        'name': 'datadisk',
+        'autoDelete': True,
+
+        # Within the docker container, specify a mount point for the disk.
+        # The pipeline input argument below will specify that inputs should be
+        # written to this disk.
+        'mountPoint': '/mnt/data',
+
+        # Specify a default size and type
+        'sizeGb': 500,            # TODO: remove this when the API has a default
+        'type': 'PERSISTENT_HDD', # TODO: remove this when the API has a default
+      } ],
+    },
+
+    # Specify the docker image to use along with the command
+    'docker' : {
+      'imageName': 'gcr.io/%s/fastqc' % args.project,
+
+      # The Pipelines API will create the input directory when localizing files,
+      # but does not create the output directory.
+      'cmd': ('mkdir /mnt/data/output && '
+              'fastqc /mnt/data/input/* --outdir=/mnt/data/output/'),
+    },
+
+    # The Pipelines API currently supports full GCS paths, along with patterns (globs),
+    # but it doesn't directly support a list of files being passed as a single input
+    # parameter ("gs://bucket/foo.bam gs://bucket/bar.bam").
```

Comment:
That is not the JES code-base. That is Cromwell's use of JES.
I have already filed a feature request with the Google Engineering team to support a list of paths.